### PR TITLE
DisplayShort version

### LINKED ISSUE #948: Display a short version of balances when it makes sense
We are getting the formatted balances from the api, but in some places we want to display a shorter version:

IMG_1014

### PR DESCRIPTION
**Pull request by Ekaanth**
fix #948

### CODE REVIEW

File: src/ui/screens/Parachains/AuctionsScreen.tsx [hunk 1]
```diff
@@ -35,6 +36,7 @@ export function AuctionsScreen() {
 
   const {auctionsInfo, latestAuction} = auction;
   const {winningBid, leasePeriod, endingPeriod, raised, raisedPercent} = latestAuction;
+  const raisedFormatted = formatBalance(raised) ?? '';
```
Comment (ishrak-imam) [inline src/ui/screens/Parachains/AuctionsScreen.tsx]: You have to pass the `isShort` argument to get the short value.

```suggestion
  const raisedFormatted = formatBalance(raised, {isShort: true}) ?? '';
```

**Review by ishrak-imam — COMMENTED** [review]
Can you also check if we need to display the short version anywhere else in the app.

**Comment by Ekaanth** [comment]
> Can you also check if we need to display the short version anywhere else in the app.
I haven't come across any pages where the value overlaps on the graphs.

**Review by silva-fj — APPROVED** [review]

**Comment by silva-fj** [comment]
> > Can you also check if we need to display the short version anywhere else in the app.
> > I haven't come across any pages where the value overlaps on the graphs.

You can just manually hard code a value to test it

**Comment by Ekaanth** [comment]
> > > Can you also check if we need to display the short version anywhere else in the app.
> > > I haven't come across any pages where the value overlaps on the graphs.
> 
> You can just manually hard code a value to test it

In all of the applications, we display tokens in only one place, and that is here.

**Review by ishrak-imam — APPROVED** [review]

**Comment by silva-fj** [comment]
> > > > Can you also check if we need to display the short version anywhere else in the app.
> > > > I haven't come across any pages where the value overlaps on the graphs.
> > 
> > 
> > You can just manually hard code a value to test it
> 
> In all of the applications, we display tokens in only one place, and that is here.

Not exactly, we display amounts in the crowdloan screen as well. In small screens in the crowdloan screen looks a bit odd when the amount is high, but I think we need to solve that issue in a different way there, it's important to display the total formatted amount

**Comment by Ekaanth** [comment]
> > > > > Can you also check if we need to display the short version anywhere else in the app.
> > > > > I haven't come across any pages where the value overlaps on the graphs.
> > > 
> > > 
> > > You can just manually hard code a value to test it
> > 
> > 
> > In all of the applications, we display tokens in only one place, and that is here.
> 
> Not exactly, we display amounts in the crowdloan screen as well. In small screens in the crowdloan screen looks a bit odd when the amount is high, but I think we need to solve that issue in a different way there, it's important to display the total formatted amount

Despite showing the formatted value we still have the issue with displaying, it has more to do with the design and can be addressed when we tackle this issue.

**Pull request merged**